src/pypulseq/calc_rf_bandwidth.py

```diff
@@ -1,17 +1,21 @@
 import math
+import warnings
 from types import SimpleNamespace
 from typing import Tuple, Union
 
 import numpy as np
 
 from pypulseq.calc_rf_center import calc_rf_center
+from pypulseq.opts import Opts
 
 
 def calc_rf_bandwidth(
     rf: SimpleNamespace,
     cutoff: float = 0.5,
     return_axis: bool = False,
     return_spectrum: bool = False,
+    dw: float = 10,
+    dt: float = 1e-6,
```

Comment:
better use the default system.rf_raster_time here instead of hardcoding a value?
probably best practice to make it float | None with default None and if None, we use the default?